### DISCUSSION TITLE
Canva Accesss

### DISCUSSION THREAD

**Discussion by Ahmad-Grewal** [community/community#65301] [GitHub Education]
### Hi everyone,

Can Anybody tell me Why i cannot Get access to Canva as a student Developer pack

**Comment by geoffreylgv** [comment]
Hi @Ahmad-Grewal, Canva (Canva Pro version) is no longer offered to students in the GH developer pack.

**Comment by CanvaProPricing** [comment]
[Canva Pro Pricing](https://github.com/CanvaProPricing/Cost/): Just $8/Month! https://github.com/CanvaProPricing/Cost/
